examples/tutorials/advanced/focal_mechanisms.py

```diff
@@ -122,8 +120,8 @@
     longitude=2,
     latitude=0,
     depth=0,
-    compressionfill="p8",
-    extensionfill="p31",
+    compressionfill=Pattern(8),
+    extensionfill=Pattern(31),
```

Comment:
Just want to flag that we need to remember to update the type-hints for various `*fill` parameters to include `pygmt.params.Pattern` later.